apps/mark/frontend/src/app_root.tsx

```diff
@@ -649,17 +650,23 @@ export function AppRoot({
   }
   if (isSystemAdministratorAuth(authStatus)) {
     return (
-      <SystemAdministratorScreen
-        logger={logger}
-        unconfigureMachine={unconfigure}
-        isMachineConfigured={Boolean(optionalElectionDefinition)}
-        resetPollsToPaused={
-          pollsState === 'polls_closed_final'
-            ? makeAsync(resetPollsToPaused)
-            : undefined
-        }
-        usbDriveStatus={usbDrive.status}
-      />
+      <React.Fragment>
```

Comment:
This file is easier to review using https://github.com/votingworks/vxsuite/pull/3217/commits/8b215a6322844b05523f248dbfaca6442ce88bad?w=1